NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.ts

```diff
@@ -13,6 +13,8 @@ import { InterviewType } from '../../models/interview-type';
 import { Router } from '@angular/router';
 import { DateService } from '../../services/date-service/date.service';
 import { DateTimePickerComponent } from '../datetimepicker/datetimepicker.component';
+import { Timestamp } from 'rxjs/internal/operators/timestamp';
```

Comment:
I don't think Timestamp, DateService, and DateTimePickerComponent are necessary to import here